curriculum/challenges/english/25-front-end-development/lecture-react-strategies-and-debugging/67d1ad82cff954a854bcbcaa.md

```diff
@@ -1,17 +1,12 @@
 ---
 id: 67d1ad82cff954a854bcbcaa
 title: What Is Prop Drilling?
-challengeType: 11
-videoId: 83LkOesFkWI
+challengeType: 19
 dashedName: what-is-prop-drilling
 ---
 
 # --description--
 
-Watch the video or read the transcript and answer the questions below.
-
-# --transcript--
-
 What is prop drilling?
 
```

Comment:
It seems we are removing the duplicate titles, so this change should be applied to all of the lectures.

```suggestion
```